tests/components/mqtt/test_select.py

```diff
@@ -609,60 +602,86 @@ async def test_entity_debug_info_message(
     )
 
 
-@pytest.mark.parametrize("options", [["milk", "beer"], ["milk"], []])
+@pytest.mark.parametrize(
+    ("hass_config", "options"),
+    [
+        (
+            {
+                mqtt.DOMAIN: {
+                    select.DOMAIN: {
+                        "command_topic": "test/select/set",
+                        "state_topic": "test/select",
+                        "name": "Test select",
+                        "options": ["milk", "beer"],
+                    }
+                }
+            },
+            ["milk", "beer"],
+        ),
+        (
+            {
+                mqtt.DOMAIN: {
+                    select.DOMAIN: {
+                        "command_topic": "test/select/set",
+                        "state_topic": "test/select",
+                        "name": "Test select",
+                        "options": ["milk"],
+                    }
+                }
+            },
+            ["milk"],
+        ),
+        (
+            {
+                mqtt.DOMAIN: {
+                    select.DOMAIN: {
+                        "command_topic": "test/select/set",
+                        "state_topic": "test/select",
+                        "name": "Test select",
+                        "options": [],
+                    }
+                }
+            },
+            [],
+        ),
```

Comment:
It looks like the options are same, please deduplicate by doing `@pytest.mark.parametrize` two times